Tsdye

tsdye opened the pull request:
Here are the slides, the original figures, and the Org mode source code for my presentation.

Commit:
first commit

Commit:
Merge branch 'tsdye' of https://github.com/tsdye/SAA2015-Open-Methods into tsdye

benmarwick commented:
Thanks very much, that's great. I reorganised a tiny bit and added a readme.md, please check to make sure it looks ok to you. Thanks again.

tsdye commented:
Hi Ben,

Looks good.  Thanks.

The slide show contains links at the end to other project-specific
material, so I think we're good to go.

I'm looking forward to meeting you and the others next week.

All the best,
Tom

Ben Marwick notifications@github.com writes:

> Thanks very much, that's great. I reorganised a tiny bit and added a
> readme.md, please check to make sure it looks ok to you. Thanks again.
> 
> ---
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/benmarwick/SAA2015-Open-Methods/pull/1#issuecomment-90683121
> Thanks very much, that's great. I reorganised a tiny bit and added a
> readme.md, please check to make sure it looks ok to you. Thanks again. 
> 
> —
> Reply to this email directly or view it on GitHub.

## 

T.S. Dye & Colleagues, Archaeologists
735 Bishop St, Suite 315, Honolulu, HI 96813
Tel: 808-529-0866, Fax: 808-529-0884
http://www.tsdye.com
